front-end/src/TermsOfService.js

```diff
@@ -94,8 +107,10 @@ function TermsOfService() {
 		<ul>
 		    <li>By visiting <Link to="/meet-the-team">our contact page.</Link></li>
 		</ul>
-		
-		<p><br></br></p>
+		<a href = "#terms-and-conditions">Back to top</a>
+		<p></p>
+		<br></br><br></br>
```

Comment:
the p tag can just be removed, correct? We don't need it

Reply:
line 111, that is

Reply:
I didn't notice those, I removed them now!